### DISCUSSION TITLE
Kicking the tires on Astroid 3 : Video : Further video suggestions?

### DISCUSSION THREAD

**Discussion by paurray** [templaza/astroid-framework#434] [Show and tell]
Hi, hi

I finally got my head around MAMP and have several Joomla test sites up and running. I made another video: Kicking the tires on Astroid 3 for Joomla 4 & 5. Maybe useful for some one?

https://youtu.be/_kxe5sgj4rI

If any one has ideas for further videos I am all ears. Am thinking hands on step by step stuff. Things I am thinking of.

A) Multilingual Site with Astroid
B) Some kind of tutorial about different menu options eg Joomla/Astroid/Mobile?

Best

Paul

**Comment by sonvnn** [comment]
Thank you @paurray ! Your video is very clear and helpful. Would you like to contribute to our blog? I will create a channel for you on Astroidframe.work. So you can feel free to post your videos on our website. We need volunteers like you.

Thank you very much

**Comment by sonvnn** [comment]
@paurray We can create a video tutorial page like this https://astroidframe.work/blog

Looking forward to hearing from you.
Best Regards,
Sonny

**Comment by paurray** [comment]
@sonvnn Let me have a little think about this and how it might work. Probably yes.
As you have no doubt gathered I am a huge Fan of all things Astroid.
Would be more like start of 2024 + though….

In the meantime would be grateful if you and or others could help me
Need to get 2 things working (In MAMP)

- Want to get up to date = Astroid 3+ (Just posted)
- Want to get latest Joomla 5 working (My SQL version challenges / will post)

Also if you or someone else could get me link from here:
https://www.youtube.com/@Astroid-Joomla-Seamlessly
to here:
https://www.youtube.com/@astroidframework2932
Similar to how I did in Screen shot below...
It would probably help.

Ditto if any one else sees this and want to link, like, subscribe etc

thanks

Paul

<img width="960" alt="Astroid   Joomla Seamlessly" src="https://github.com/templaza/astroid-framework/assets/9163424/db8e1cc8-82fa-4c84-8817-00350124ed4c">

**Reply by sonvnn** [reply]
Please register an account on astroidframe.work. I will set your permission for you. So you can create video post link to your youtube channel.

**Comment by sonvnn** [comment]
@paurray Please send me your email via sonlv@templaza.com. I will help you create an account on astroidframe.work
After that you can post your video to Astroid Blog.

Thanks & Best regards,
Sonny

**Comment by Chacapamac** [comment]
@paurray   @sonvnn  — This is great news!
It will be great if these videos appear or replace the old videos in the Astroid YouTube Channel
https://www.youtube.com/channel/UCUHl1uU0Ofkyo-1ke-K4_xg

**Remark:** I think the URL of the channel _UCUHl1uU0Ofkyo-1ke-K4_xg_ should be change for something more Pro like:
https://www.youtube.com/channel/AstroidFramework

**Comment by paurray** [comment]
@Chacapamac @sonvnn 
Both of these links take me to the Astroid Framework @astroidframework2932 Channel

https://www.youtube.com/channel/UCUHl1uU0Ofkyo-1ke-K4_xg
https://www.youtube.com/@astroidframework2932

I think it has to do with from where exactly you are copying the channel link from

Maybe this helps tidy things up:

How to change youtube URL 2023 – IN 1 MINUTE!
https://www.youtube.com/watch?v=k8w1SWq8QDI

With respect to Adding linking up Channels with Channels…

Something like this:

How To Feature Other Channels On YouTube 2023 (YouTube Channel Featured Section Video)
https://www.youtube.com/watch?v=-PIzPk6OxZw

**Comment by Chacapamac** [comment]
@paurray 
The https://www.youtube.com/@astroidframework2932 is probably the Public name of Astroid Channel — I didn’t know. Great, At least Astroid is there if somebody search the term “Astroid” in Youtube

• Somebody have the https://www.youtube.com/Astroid Channel with nothing in it?????

Nobody have —> https://www.youtube.com/astroidframework will be even better that —> https://www.youtube.com/@astroidframework2932

I don’t know if you can rename the channel again, not really important....

**Comment by paurray** [comment]
@Chacapamac 
I do not have and do not want access to the @astroidframework2932 site.
I guess that is a matter for @sonvnn
But I agree it makes sense to grab:
https://www.youtube.com/@astroidframework
Because
https://www.youtube.com/@astroidframework2932
Seems like an unhappy and unnecessary compromise!

Personally I am very happy with:
https://www.youtube.com/@Astroid-Joomla-Seamlessly
As it is very much my focus :-)

Again in case it was missed:
How to change youtube URL 2023 – IN 1 MINUTE!
https://www.youtube.com/watch?v=k8w1SWq8QDI

**Comment by sonvnn** [comment]
Thank you @paurray for the first article on Astroid https://astroidframe.work/blog/kicking-the-tires-on-astroid-3-for-joomla-4-5 . Keep up your great work :)

**Comment by Chacapamac** [comment]
Great article & Video @paurray  — Great Job!

**Comment by paurray** [comment]
@sonvnn @Chacapamac 
Thanks for the positive feedback
Really helps the old motivation :-)
Am thinking of starting at the start again next year with basic install videos.
Feed back appreciated here:
https://github.com/templaza/astroid-framework/discussions/487
Or other suggestions?
